src/scripts/install.sh

```diff
@@ -1,8 +1,5 @@
 # shellcheck disable=SC2148
-
-set -x
 ORB_STR_AWS_CLI_VERSION="$(echo "${ORB_STR_AWS_CLI_VERSION}" | circleci env subst)"
```

Comment:
Unless you have a reason not to, change the line below to match the usage of `circleci env subst` in other scripts in this repo.

```suggestion
ORB_STR_AWS_CLI_VERSION="$(circleci env subst ${ORB_STR_AWS_CLI_VERSION})"
```